<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>结束条件</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>最大时间(分)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBox_2"/>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spbMaxItem"/>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="chkMaxItem">
           <property name="text">
            <string>最多物件</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>111</height>
         </size>
        </property>
        <property name="title">
         <string>SQL</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>hostname</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>password</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>导出设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>jl</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>json</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_4">
           <property name="text">
            <string>调试模式</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QLineEdit" name="lineEdit_3">
           <property name="placeholderText">
            <string>images</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>导出路径</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>CSV</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_5">
           <property name="text">
            <string>mongo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>xls</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="4">
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="chbAddScratch">
           <property name="text">
            <string>增量抓取</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButton_6">
           <property name="text">
            <string>sql</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="checkBox_5">
           <property name="text">
            <string>记录模式</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>455</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>漫画</string>
        </attribute>
        <widget class="QLineEdit" name="ledBookname">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>130</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>输入书名拼音，多本书使用分号间隔</string>
         </property>
         <property name="placeholderText">
          <string>yaojingdeweiba</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>10</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>动漫之家</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>87</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>dmzj</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>sgacg</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>acqq</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>https://manhua.dmzj.com/</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>BBS</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>页</string>
        </attribute>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>70</y>
           <width>301</width>
           <height>191</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>490</y>
           <width>72</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ledRunstring">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>340</y>
           <width>221</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="btnRun">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>330</y>
           <width>81</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string>run</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>表格</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="textBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>123456</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>9789796</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../../../../../../works/zalSer/code/ui/eSerial.qrc">
     <normaloff>:/img/开始_dis.png</normaloff>:/img/开始_dis.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../../../../../works/zalSer/code/ui/eSerial.qrc">
     <normaloff>:/img/stop_326px_1187740_easyicon.net.png</normaloff>:/img/stop_326px_1187740_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="shortcut">
    <string>Shift+F5</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../../../../../../works/zalSer/code/ui/eSerial.qrc">
     <normaloff>:/img/editclear_128px_32658_easyicon.net.png</normaloff>:/img/editclear_128px_32658_easyicon.net.png</iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pyscr.qrc"/>
 </resources>
 <connections/>
</ui>
